src/OpenSpaceNetArgs.cpp

```diff
@@ -141,38 +141,38 @@ OpenSpaceNetArgs::OpenSpaceNetArgs() :
         ;
 
     processingOptions_.add_options()
-        ("cpu", "Use the CPU for processing, the default it to use the GPU.")
+        ("cpu", "Use the CPU for processing, the default is to use the GPU.")
         ("max-utilization", po::value<float>()->value_name(name_with_default("PERCENT", maxUtilization)),
          "Maximum GPU utilization %. Minimum is 5, and maximum is 100. Not used if processing on CPU")
         ("model", po::value<string>()->value_name("PATH"), "Path to the the trained model.")
-        ("window-size", po::cvSize_value()->min_tokens(1)->value_name("WIDTH [HEIGHT]"),
-         "Overrides the original model's window size. Window size can be specified in either one or two dimensions. If "
-         "only one dimension is specified, the window will be square. This parameter is optional and not recommended.")
+        ("window-sizes", po::value<std::vector<int>>()->multitoken()->value_name("STEP [STEP...]"),
+         "Sliding window detection box sizes.  The source image is chipped with boxes of the given sizes.  "
+         "Default is a single window with the model size.")
+        ("window-steps", po::value<std::vector<int>>()->multitoken()->value_name("SIZE [SIZE...]"),
+         "Sliding window step.  Either a single step or a step for each window size may be given.  Default value is 20% of the model size.")
+        ("resampled-size", po::value<int>()->value_name("SIZE"),
```

Comment:
Why is this resampled-size? Are we confusing the arguments again. I thought this was supposed to be the "override size", meaning add border, meaning no resampling....